blog/2022/04/simplechains.md

```diff
@@ -0,0 +1,493 @@
+
+However, in this post we'll take a different look at how the programmability of Julia can help in the machine learning space. Specifically, by targetting the grand space of "all machine learning", the frameworks inevitably make trade-offs that accelerate some aspects of the code at the detriment to others. This comes from the inevitable trade-off between simplicity, generality, and performance. However, the ability to easily construct machine learning libraries thus presents an interesting question: can this development feature be used to easily create alternative frameworks which focus its performance on more non-traditional applications or aspects?
+
+The answer is yes, you can quickly build machine learning frameworks which greatly outperform the giants in specialized cases using the Julia programming language, and we demonstrate this with our new release [SimpleChains.jl](https://github.com/PumasAI/SimpleChains.jl).
+
+#### Note before we start
+
+If you're interested in this topic and want to work on Julia machine learning, note that the [Pumas DeepPumas team is hiring](https://pumas.ai/company/machine-learning-scientist/). Additionally, [internships are available at Julia Computing](https://jobs.juliacomputing.com/jobs/ndvlJz9fHYcr/machine-learning-intern-remote).
+
+## Scientific Machine Learning (SciML) and "Small" Neural Networks
+
+SimpleChains.jl is a library developed by [Julia Computing](https://juliacomputing.com/) and [Pumas-AI](https://pumas.ai/) in collaboration with [Roche](https://www.roche.com/) and the [University of Maryland, Baltimore](https://www.pharmacy.umaryland.edu/centers/ctm/). The purpose of SimpleChains.jl is to be as fast as possible for small neural networks. This is because SimpleChains.jl originated as a solution for the DeepPumas product for [scientific machine learning (SciML)](https://www.stochasticlifestyle.com/the-essential-tools-of-scientific-machine-learning-scientific-ml/) in clinical pharmacology. In this domain, small neural networks are combined with known nonlinear mixed effects models (statistical models with differential equations) to discover previously unknown mechanisms and prognostic factors. For a short introduction to how this is done, check out the following video by Niklas Korsbo:
```

Comment:
This is because SimpleChains.jl originated as a solution for the DeepPumas product for [scientific machine learning (SciML)](https://www.stochasticlifestyle.com/the-essential-tools-of-scientific-machine-learning-scientific-ml/) in healthcare data analytics.